pages/jhipster-lite.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: JHipster Lite
+permalink: /jhipster-lite/
+sitemap:
+    priority: 0.7
+    lastmod: 2022-09-14T00:00:00-00:00
+---
+
+# What is JHipster Lite?
+
+As JHipster, **JHipster Lite ⚡** will help you to start your project, focusing on generating step by step only what you need.
+
+Some details about the solution:
+- The generated code uses [Hexagonal Architecture]
```

Comment:
we can add the reference to this doc ?
https://github.com/jhipster/jhipster-lite/blob/main/documentation/hexagonal-architecture.md rather than the schema